tgnnexplainer/xgraph/evaluation/metrics_tg.py

```diff
@@ -150,13 +160,15 @@ class EvaluatorMCTSTG(BaseEvaluator):
     def __init__(self, 
         model_name: str, explainer_name: str, dataset_name: str, 
         explainer: SubgraphXTG,
-        results_dir = None
+        results_dir = None,
+        threshold_num=25
         ) -> None:
         super(EvaluatorMCTSTG, self).__init__(model_name=model_name,
                                               explainer_name=explainer_name,
                                               dataset_name=dataset_name,
                                               results_dir=results_dir,
-                                            #   explainer=explainer
+                                              # explainer=explainer,
```

Comment:
Why is this commented out?

Reply:
I assume because this class defines self.explainer anyway. It was commented out by the authors. Didn't want to touch it